gen.go

```diff
@@ -471,9 +478,9 @@ func NewObjectPropertyMappingValue(base string, named *types.Named, name string,
 				exportedFieldType: f.Type(),
 			}
 			// proto generated Getter
-			getter, ok := GetMethod(named, "Get"+name, ignoreCase)
-			if ok && getter.Exported() {
-				ret.getter = getter
+			getterMethod, ok := GetMethod(named, "Get"+name, ignoreCase)
+			if ok && getterMethod.Exported() {
+				ret.getterMethod = getterMethod
```

Comment:
I added a new seperate field to store the getter method, which will take precedent when populating the source value in the `genAssignmentStmt` func. This feels a bit hacky but was pretty unintrusive compared to other options, since we plan on moving to the main upstream in the future felt it wouldn't be too bad.

Let me know if there's a better approach.